blueprintapi/urls.py

```diff
@@ -14,10 +14,23 @@
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
 from django.contrib import admin
-from django.urls import include, path
+from django.urls import include, path, re_path
+from rest_framework import permissions
+from drf_yasg.views import get_schema_view
+from drf_yasg import openapi
 
 from . import views
 
+schema_view = get_schema_view(
+    openapi.Info(
+        title="RapidATO API",
+        default_version='v1',
+        description="Welcome to the world of RapidATO",
```

Comment:
For some reason, my brain stopped reading this sentence part way through and changed the sentence to "Welcome... to Jurassic Park"